.github/workflows/bleeding-edge.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   test:
+    if: github.repository == 'enthought/traitsui'
```

Comment:
We could test `repository_owner` instead, for a version that's more copy-pastable between Enthought repos.